web-client/src/app/views/wallet-access/wallet-access.page.ts

```diff
@@ -46,26 +42,25 @@ export class WalletAccessPage implements OnInit {
     const address = value?.trim();
 
     if (address && isValidAddress(address)) {
-      const pinPromise = this.presentLock();
+      const pinPromise = await this.presentLock();
```

Comment:
Double check that the loading control gets shown correctly on mobile. I ran into an issue where there is downtime between canceling or entering the pin, and when the loading indicator displays, so that is why the promise gets awaited later.

Reply:
I will do that and report back

Reply:
@longtomjr I haven't experienced any issue with the loading indicator